duckduckgo/BrowserServicesKit
# SwiftLint fixes part 3

Please review the release process for BrowserServicesKit [here](https://app.asana.com/0/1200194497630846/1200837094583426).

**Required**:

Task/Issue URL: https://app.asana.com/0/0/1205234591569346/f
iOS PR: No changes needed, please drop BSK onto the iOS repo to test
macOS PR: No changes needed, please drop BSK onto the iOS repo to test
What kind of version bump will this require?: Patch, no functional changes are made

**Optional**:

Tech Design URL:
CC:

**Description**:

This PR fixes all remaining SwiftLint violations.



**Steps to test this PR**:
1. Check that CI passes
1. Check that `swiftlint .` passes when you have this branch checked out locally, as it will lint all files



**OS Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
